src/pyclass.jl

```diff
@@ -160,9 +168,17 @@ Multiple dispatch works, too:
     x2.set!(self, new_x::Int) = ...
     x2.set!(self, new_x::Float64) = ...
 """
-macro pydef(type_expr)
-    type_name, base_classes, methods_, getter_dict, setter_dict, function_defs =
-        parse_pydef(type_expr)
+macro pydef(class_expr)
+    class_name, _, _ = parse_pydef_toplevel(class_expr)
+    :($(esc(class_name)) = @pydef_object($(esc(class_expr))))
```

Comment:
Probably should be `const class_name = ...`, since types in Julia are usually `const`